recipes/aws-noaa-whoi/recipe.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+start_date = '1988-01-01'
+
+
+def format_function(time):
+    base = pd.Timestamp(start_date)
+    day = base + pd.Timedelta(days=time)
+    input_url_pattern = (
+        's3://noaa-cdr-sea-surface-temp-whoi-pds/data/{day:%Y}'
+        '/SEAFLUX-OSB-CDR_V02R00_SST_D{day:%Y%m%d}_C*.nc'
```

Comment:
Does this actually work? I did not think glob-style wildcards were supported by s3fs! 🤯

Reply:
Implemented deterministic file pattern as recommended above and successfully tested on a 3 year dataset. All output looks good.